src/jaxsim/rbda/actuation/common.py

```diff
@@ -0,0 +1,11 @@
+import jaxsim.typing as jtp
+from jaxsim.utils import JaxsimDataclass
+
+class ActuationParams(JaxsimDataclass):
+    """
+    Abstract class representing the parameters of an actuation model.
```

Comment:
This is not an abstract class
```suggestion
    Parameters class for the actuation model.
```